nsqadmin/static/js/views/app.js

```diff
@@ -136,6 +136,10 @@ var AppView = BaseView.extend({
     },
 
     onLinkClick: function(e) {
+        if (e.ctrlKey || e.metaKey) {
+          // allow ctrl+click to open in a new tab
+          return true
+        }
         e.preventDefault();
         e.stopPropagation();
```

Comment:
If you were stopping the event propagation so that the click isn't registered on another element, I'm guessing you'll still want to do that even when ctrl+clicking. Maybe move the `stopPropagation` above the `if` clause?